osquery/events/linux/conntrack.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <sstream>
+
+#include <fnmatch.h>
+#include <linux/limits.h>
+#include <poll.h>
+
+#include <arpa/inet.h>
+
+#include <libmnl/libmnl.h>
+#include <libnetfilter_conntrack/libnetfilter_conntrack.h>
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+
+#include "osquery/events/linux/conntrack.h"
+
+namespace osquery {
+
+/// The conntrack subsystem may have a performance impact on the system.
+FLAG(bool,
+     disable_conntrack,
```

Comment:
If this is default-disabled, let's rename to `enable_conntrack` and set the default to `false`. Then if folks want the capability they'll use: `--enable_conntrack`.